server/reload_test.go

```diff
@@ -11,6 +11,7 @@ import (
 	"time"
 
 	"github.com/nats-io/go-nats"
+	"net"
```

Comment:
Nitpick: package import ordering should be:

1. stdlib
2. third-party
3. local